actions/create_document.lua

```diff
@@ -1,27 +1,29 @@
 event: ["request_post_document"]
 priority: 1
 
-local helpers = require("helpers")
+local content = require("content")
 
 -- POST /
 local post_uuid = uuid.v4()
-local file = io.open("content/" .. post_uuid, "w")
+local file_uuid = uuid.v4()
+
 local params = {
-    title = request.body.title,
     type = request.body.type,
+    uuid = post_uuid,
+    attributes = {
```

Comment:
can this be called fields?

Reply:
You would like attributes to be called fields?

Reply:
yes